packages/service/lib/stackable.js

```diff
@@ -94,6 +94,10 @@ class ServiceStackable {
 
     this.app.log[logLevel](message)
   }
+
+  async getBootstrapDependencies () {
+    return []
+  }
```

Comment:
I think we could have this as "optional" and not require an empty implementation.